### PR TITLE
Add support for firefox installed with flatpak

### PR DESCRIPTION
**Pull request by codename-cn**
I  added install support for firefox installed with flatpak.

### CODE REVIEW

File: tweaks.sh [hunk 1]
```diff
@@ -67,7 +67,13 @@ while [[ $# -gt 0 ]]; do
           edit_firefox="true" ;;
       esac
 
-      if ! has_command firefox; then
+      if [[ -d "${FIREFOX_FLATPAK_DIR_HOME}" ]]; then
+          FIREFOX_DIR_HOME=${FIREFOX_FLATPAK_DIR_HOME}
```
Comment (rivanfebrian123) [inline tweaks.sh:71]: Thanks for your PR, but, it's a bad practice to change the environment variable like this

File: tweaks.sh [hunk 2]
```diff
@@ -67,7 +67,13 @@ while [[ $# -gt 0 ]]; do
...
+	  mkdir -p ${FIREFOX_FLATPAK_THEME_DIR}
```
Comment (rivanfebrian123) [inline tweaks.sh:72]: And, force making a folder in argument parsing?

**Review by rivanfebrian123 — CHANGES_REQUESTED** [review]
Once again, thanks for your PR. Keep going! :)

File: tweaks.sh (hunk 1 above)
Comment (codename-cn) [reply tweaks.sh:71]: Thanks for your feedback. Do you have a better idea to solve it?

File: tweaks.sh (hunk 2 above)
Comment (codename-cn) [reply tweaks.sh:72]: Where to put this? There is no folder "firefox-theme" on fresh install.

Comment (rivanfebrian123) [reply tweaks.sh:72]: Go explore `lib-core.sh` and `lib-install.sh`, and take a look the `install_firefox_theme` function. You better to put your code there.

Oh yeah, also, please take a look at `install_dash_to_dock_theme` and imitate the way we switch the correct path for Dash to Dock installation.

Anyways, we're currently freezing our repo for some refactoring. You can come back and recheck the code tomorrow

File: tweaks.sh (hunk 1 above)
Comment (rivanfebrian123) [reply tweaks.sh:71]: Just keep it there :D

**Comment by rivanfebrian123** [comment]
@vinceliuice he doesn't active for a week for some reason. I think we need to close or merge this PR to a branch other than "master"

**Pull request merged**